tests/inspect/inspect.go

```diff
@@ -336,7 +336,7 @@ func main() {
 	}
 
 	if globalFlags.CheckCgroupMounts {
-		rootCgroupPath := "/proc/1/root/sys/fs/cgroup"
+		rootCgroupPath := "/sys/fs/cgroup"
```

Comment:
We are going to revert this change in next commit. 
That change was caused by my misunderstanding of CGroup isolation test. I've thought that this test is checking whether are cgroups mounted in ro mode on stage 2. Testing cgroups on pod make no sense in my opinion for kvm(which has got own cgroups) - so I am going to simply skip that test